wandb/integration/lightning/fabric/logger.py

```diff
@@ -0,0 +1,764 @@
+import os
```

Comment:
Isn't the implementation in this file based on the one from Lightning?
https://github.com/Lightning-AI/pytorch-lightning/blob/master/src/lightning/pytorch/loggers/wandb.py

IMO there should be at least a comment here attributing that the code was taken from there and modified.

Reply:
Hey @awaelchli. Good callout. What would you propose is a good attribution message?

Reply:
No problem. Normally the procedure is to retain the copyright notice that is [at the top of the file](https://github.com/Lightning-AI/pytorch-lightning/blob/a6273d17870336886432d7f79900a07fe7f872a6/src/lightning/pytorch/loggers/wandb.py#L1-L13). Then, if modifications are made you would add your own copyright notice.

But for simplicity you could also shorten it. For example:

```
# Copyright 2024 Lightning AI. Apache License 2.0
# Copyright 2024 Weights and Biases, Inc. MIT License
```